theme/src/components/code.js

```diff
@@ -28,6 +28,8 @@ function Code({className, children, live, noinline}) {
         {({className, style, tokens, getLineProps, getTokenProps}) => (
           <BorderBox as="pre" className={className} mt={0} mb={3} p={3} border={0} style={{...style, overflow: 'auto'}}>
             {/* This is the scroll handle, it is supposed to be focused with keyboard and scroll a wide codebox horizontally */}
+            {/* FIXME: only adding this line temporarily so it shows up in the PR diff. I'm not sure how to correctly label this semantically. */}
+            {/* eslint-disable-next-line github/a11y-role-supports-aria-props */}
```

Comment:
@github/accessibility-reviewers can I get a review on this PR, specifically this area?

Here is a page on the live site that has some of these code blocks rendered on it: https://docs.npmjs.com/about-scopes

Reply:
Would it make sense to put this in a semantic `<code>`

Reply:
I think so. If I do that, is it ok to use `visibility: hidden` and remove the `aria-label`? That what I implemented in my latest commit.  

There are no longer any lint warnings and the functionality seems the same to me, but is it still accessible?